java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -4154,6 +4154,7 @@ void testContiguousSplitWithStringsChunked() {
     }
   }
 
+  @Disabled("Test is disabled until bug #20864 is fixed")
```

Comment:
Ouch. This seems like a gnarly, long-unresolved issue. Would you be willing to take on that task?